assets/css/style.css

```diff
@@ -154,11 +154,9 @@ img {
     border: 0;
 }
 
-a,
-a:hover,
-a:focus {
+a {
```

Comment:
Since we are breaking from the theme style with `var(--primary-color)`, could you add a one-line comment explaining this change, or add the link to the GitHub issue for future reference? Perhaps we will want to change the color scheme later, so it will be good to note that this was a one-off decision made for accessibility purposes.